test/contracts/IncentivePool/update.test.ts

```diff
@@ -0,0 +1,250 @@
+      expect(expectedIncentiveIndexAfterUpdate).to.be.equal(incentivePoolDataAfter.incentiveIndex);
+      expect(await elyfiContracts.incentivePool.amountPerSecond()).to.be.equal(
+        newIncentiveAmountPerSecond
+      );
+    });
+
+    it('reverts if general account0 update incentive amount per second', async () => {
+      const newIncentiveAmountPerSecond = testIncentiveAmountPerSecond.mul(2);
+
+      await expect(
+        elyfiContracts.incentivePool
+          .connect(account0)
+          .setAmountPerSecond(newIncentiveAmountPerSecond)
+      ).to.be.revertedWith('onlyAdmin');
+    });
+
+    it('admin can update endTimestamp', async () => {
+      const newEndTimestamp = (await elyfiContracts.incentivePool.endTimestamp()).add(1);
+
+      const tx = await elyfiContracts.incentivePool
+        .connect(deployer)
+        .setEndTimestamp(newEndTimestamp);
+
+      expect(tx)
+        .to.be.emit(elyfiContracts.incentivePool, 'IncentiveEndTimestampUpdated')
+        .withArgs(newEndTimestamp);
+      expect(await elyfiContracts.incentivePool.endTimestamp()).to.be.equal(newEndTimestamp);
+    });
+
+    it('reverts if general account0 update endTimestamp', async () => {
+      const newEndTimestamp = (await elyfiContracts.incentivePool.endTimestamp()).add(1);
+
+      await expect(
+        elyfiContracts.incentivePool.connect(account0).setEndTimestamp(newEndTimestamp)
+      ).to.be.revertedWith('onlyAdmin');
+    });
+  });
+
+  context('update incentivePool', async () => {
+    let newIncentivePool: IncentivePool;
+    let newIncentivePoolInitTimestamp: BigNumber;
+
+    beforeEach('', async () => {
+      const incentivePoolFactory = (await ethers.getContractFactory(
+        'IncentivePool'
+      )) as IncentivePool__factory;
+
+      newIncentivePool = await incentivePoolFactory.deploy(
+        deployer.address,
+        elyfiContracts.incentiveAsset.address,
+        testIncentiveAmountPerSecond
+      );
+
+      await elyfiContracts.underlyingAsset
+        .connect(deployer)
+        .transfer(account0.address, utils.parseEther('1000'));
+      await elyfiContracts.underlyingAsset
+        .connect(deployer)
+        .transfer(account1.address, utils.parseEther('1000'));
+      await elyfiContracts.incentiveAsset
+        .connect(deployer)
+        .transfer(elyfiContracts.incentivePool.address, RAY);
+
+      await elyfiContracts.underlyingAsset
+        .connect(account0)
+        .approve(elyfiContracts.moneyPool.address, RAY);
+      await elyfiContracts.underlyingAsset
+        .connect(account1)
+        .approve(elyfiContracts.moneyPool.address, RAY);
+
+      await elyfiContracts.moneyPool
+        .connect(account1)
+        .deposit(elyfiContracts.underlyingAsset.address, account1.address, amount);
+
+      await elyfiContracts.moneyPool
+        .connect(account0)
+        .deposit(elyfiContracts.underlyingAsset.address, account0.address, amount);
+
+      await elyfiContracts.moneyPool
```

Comment:
@donguks 
After deploying, execute updateIncentivePool in moneyPool contract to update new incentivePool. Args should be like below.